src/profile-logic/call-node-info.js

```diff
@@ -201,4 +192,893 @@ export class CallNodeInfoImpl implements CallNodeInfo {
 
     return null;
   }
+
+  getRoots(): IndexIntoCallNodeTable[] {
```

Comment:
Oh yeah I noticed that too! The reason it's uncovered is because the code is actually unused! Nothing ends up calling getRoots() and getChildren() on the non-inverted call node info at the end of this PR. Those methods are only called on the inverted call node info. I suppose I could move the method declarations to the `CallNodeInfoInverted` interface but it seems a bit unclean - these methods aren't really specific to the inverted case, it's just an odd circumstance that they're not called on in the non-inverted case.

Reply:
I wrote this in another comment too, but I think we just don't call getRoots() and getChildren() on the non-inverted call node info.

Reply:
I've removed `getRoots()` and `getChildren()` from the `CallNodeInfo` interface, and removed the unused implementations on the non-inverted call node info.